calicoctl/calicoctl/commands/version.go

```diff
@@ -56,6 +56,8 @@ Options:
      --poll=<POLL>             Poll for changes to the cluster information at a frequency specified using POLL duration
                                (e.g. 1s, 10m, 2h etc.). A value of 0 (the default) disables polling.
      --allow-version-mismatch  Allow client and cluster versions mismatch.
+	 --client                  Display the client version only.
```

Comment:
Indentation looks off here

Reply:
Oh, it looks good in my vscode, let me recheck it.